packages/astro/src/runtime/server/astro-island-styles.ts

```diff
@@ -0,0 +1,2 @@
+// NOTE: if you change the value inside the style tag, you must update `prebuild.js` too
```

Comment:
Could the script be updated to read the styles from here instead?

Reply:
Sure, we can do anything